rendermodules/rough_align/schemas.py

```diff
@@ -50,22 +56,41 @@ class ApplyRoughAlignmentTransformParameters(RenderParameters):
         default=10,
         missing=10,
         metadata={'description':'pool size for parallel processing'})
-    minZ = mm.fields.Int(
-        required=True,
-        metadata={'description':'Minimum Z value'})
-    maxZ = mm.fields.Int(
+    new_z = List(
+        Int,
+        required=False,
+        default=[-1],
+        cli_as_single_argument=True,
+        description="List of new z values to be mapped to")
+    old_z = List(
+        Int,
         required=True,
-        metadata={'description':'Maximum Z value'})
+        cli_as_single_argument=True,
+        description="List of z values to apply rough alignment to")  
+    
 
     @post_load
     def validate_data(self, data):
         if data['prealigned_stack'] is None:
             data['prealigned_stack'] = data['montage_stack']
         if data['map_z']:
-            if data['map_z_start'] == -1:
-                raise ValidationError("map_z_start is invalid")
+            if data['new_z'][0] == -1:
```

Comment:
Is there a reason to have this default to `[-1]`?  `None` or missing might be more transparent for use cases which have negative numbers.

Reply:
perhaps it was to avoid exceptions when trying to access those keys? I could suggest this instead..

```
if data['map_z']:
    try:
        if abs(len(data['new_z']) - len(data['old_z'])) != 0:
            raise ValidationError("new_z list count does not match with old_z list count")
    except KeyError:
            raise ValidationError("new_z and old_z must be specified if map_z is True")
```